template/module/readme/CONFIGURE.rst

```diff
@@ -5,6 +5,6 @@ To configure this module, you need to:
 
 #. Go to ...
 
-.. figure:: path/to/local/image.png
+.. figure:: ../static/description/image.png
```

Comment:
Maybe this should be ../static/images for being better classified.

Reply:
For me, the objective of this commit is to give an exemple with correct relative path (including ../)

I abstain on that point, feel free to make the change directly on my PR if you think it's relevant.

Just to know, is it a new OCA conventions ?

FYI : 
``find ./ -name 'description' -type d -printf '.' | wc -c`` 

> 3905

``find ./ -name 'images' -type d -printf '.' | wc -c``

> 402

(source : All OCA code source, including OCB, OpenUpgrade, ...)

Reply:
No, it's not any guideline, just some random thoughts, as with this we don't need to be in that folder. @sbidoul what do you think?